source/MRMesh/MRPointCloud.h

```diff
@@ -24,6 +24,10 @@ struct PointCloud
     // if toWorld transformation is given then returns minimal bounding box in world space
     MRMESH_API Box3f computeBoundingBox( const AffineXf3f * toWorld = nullptr ) const;
 
+    /// appends points (and normals if it possible) (from) in addition to this points
+    /// if this obj have normals and from obj haven't it then don't do anything
+    MRMESH_API void addPartByMask( const PointCloud& from, const VertBitSet& fromVerts );
```

Comment:
It maby should return mask from old indices to new

Reply:
added as parameter for uniformity with other addPartByMask methods